functions/import_summary_data.R

```diff
@@ -0,0 +1,49 @@
+#' @title Import and clean School Summary Statistics data
+#' 
+#' @description This function reads in the desired sheet of the Summary
+#' Statistics dataset and cleans variable names.
+#'
+#' @param sheet_name Name of data sheet to import
+#' @param calendar_year Year of dataset to import in calendar year format; 
+#' e.g. 2020.
+#'
+#' @return Tibble of cleaned data for given dataset and year.
+#' 
+#' @export
+#'
+#' @examples
+#' import_summary_data("LA and SCOT", 2021)
+#' import_summary_data("LA and SCOT", "timeseries")
+
+import_summary_data <- function(sheet_name, calendar_year) {
+  
+  # Define pipe
+  `%>%` <- magrittr::`%>%`
+  
+  filepath <- here::here("data", 
+                          "school_summary_statistics", 
+                          paste0(calendar_year, 
+                                 "_school_summary_statistics.xlsx"))
+  
+  # Check datafile exists and if not produce error
```

Comment:
again I would need more clarification on these comments please

Reply:
I've expanded this too but let me know if still not clear